lighthouse-core/scripts/legacy-javascript/run.js

```diff
@@ -174,13 +174,17 @@ function makeSummary(legacyJavascriptFilename) {
   let totalSignals = 0;
   const variants = [];
   for (const dir of glob.sync('*/*', {cwd: VARIANT_DIR})) {
-    /** @type {Array<{signals: string[]}>} */
+    /** @type {import('../../audits/legacy-javascript.js').Item[]} */
     const legacyJavascriptItems = require(`${VARIANT_DIR}/${dir}/${legacyJavascriptFilename}`);
-    const signals = legacyJavascriptItems.reduce((acc, cur) => {
-      totalSignals += cur.signals.length;
-      return acc.concat(cur.signals);
-    }, /** @type {string[]} */ ([])).join(', ');
-    variants.push({name: dir, signals});
+
+    const signals = [];
```

Comment:
#10867 should have failed this test in CI .... fixing here

Reply:
oh this is a thorny problem. we don't know when to run these tests because something they *depend on* changed :/ 

I don't have any great ideas to fix this though other than catching it the next time. you?

Reply:
but `test-legacy-javascript.sh` checks for any file with `legacy-javascript` in the path.

Reply:
well then that's even more alarming :)

my point is still a problem I don't know a good way to deal with, but perhaps less urgent than outright broken testing 😆